connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectCluster.java

```diff
@@ -17,10 +17,11 @@
 package org.apache.kafka.connect.util.clusters;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import org.apache.kafka.connect.cli.ConnectDistributed;
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6342 instead. Thanks!

Reply:
#6342 was merged. This PR is now rebased on top of it.